public/app/plugins/panel/xychart/XYChartPanel.tsx

```diff
@@ -51,8 +51,8 @@ export const XYChartPanel: React.FC<XYChartPanelProps> = ({
   }
 
   return (
-    <GraphNG
-      data={frames}
+    <TimeSeries
```

Comment:
The naming here is pointing to something obviously wrong with our strategy :)

Reply:
sure, now it's obvious :D

Reply:
TimeSeries? This is semantically wrong :P This should use GraphNG

Reply:
GraphNG is also wrong, it's meant to be a time-as-x / aligned wrapper for TimeSeries, Timeline, Heatmap, Horizon, OHLC/Candlestick. this needs to be another wrapper that accepts scatter data with numeric, un-aligned x/y.

Reply:
we'll tackle this in a followup